### ISSUE TITLE
Create layouts

### ISSUE THREAD

**Issue by vaibhavchellani** [GDGBVCOE/NoQueue#7]
i am attaching the rough layouts , please make the layouts , the adapters , the interfaces . Create the whole flow of the app then we shall attach it with database for backend .
All the layouts are just rough description of what should be on the page , suggestions and critisism are welcome .

**Comment by vaibhavchellani** [comment]
![1-0_home](https://user-images.githubusercontent.com/19390504/30036480-610b8832-91d1-11e7-8e10-182c3688c486.png)
![2-0_sidebar](https://user-images.githubusercontent.com/19390504/30036481-611545e8-91d1-11e7-808d-fe5add39c31d.png)
![4-0_createqueue](https://user-images.githubusercontent.com/19390504/30036482-6116eb64-91d1-11e7-9c54-fe5141dab29e.png)
![3-0_queuescreen](https://user-images.githubusercontent.com/19390504/30036483-61195ebc-91d1-11e7-8069-1bd71c6df9a9.png)

**Comment by vaibhavchellani** [comment]
@ja

**Comment by vaibhavchellani** [comment]
@IshaAgarwal98 @Jasmeet1709 @Rikki407 @Mridul97 , also suggest a colour scheme , lets discuss the screens first then i shall distribute screens to each one of you . Thanks !

**Comment by vaibhavchellani** [comment]
guys you have to acknowledge the issues

**Comment by ghost** [comment]
instead of next to next we should show previous token  in the queue screen

**Comment by vaibhavchellani** [comment]
why do you think so ?

**Comment by vaibhavchellani** [comment]
try fasthub app its the best github client for android

**Comment by ghost** [comment]
because as a person I will be interested in the token which is after and before mine

**Comment by vaibhavchellani** [comment]
what information does the previous token provide

**Comment by ghost** [comment]
also the layout of queue can a bit different as in just change in position
users  token should be should be shown then the 3 (next current and previous)

**Comment by vaibhavchellani** [comment]
yes that can be done

**Comment by ghost** [comment]
the 3 options should be 
1.current token 
2.previous token(token before user's token)
3.next token in the queue

**Comment by Jasmeet1709** [comment]
@vaibhavchellani their is no requirement of previous and next token as their is serial wise token numbers.

**Comment by vaibhavchellani** [comment]
@jasmeet1709 tokens can be cancelled also , so tokens may not be serial wise  ,right ?

**Comment by Jasmeet1709** [comment]
@vaibhavchellani okay then it is fine with only current and next.

**Comment by vaibhavchellani** [comment]
yes , should we have 3 recycler views on home screen or a view pager/ bottom navigation on the home screen ?

**Comment by vaibhavchellani** [comment]
@rikki407 please join

**Comment by Jasmeet1709** [comment]
@vaibhavchellani yeah it looks good and we need only two recycler view queues and my queues as queues created by user should be shown in the queue we select

**Comment by vaibhavchellani** [comment]
yes  right

**Comment by Mridul97** [comment]
queues created by the user gives admin right to the user so it should be shown in a different recycler view

**Comment by vaibhavchellani** [comment]
there is a tab in side bar called admin panel for managing queues the user has created I guess that should be enough

**Comment by ghost** [comment]
are we using view pager?

**Comment by vaibhavchellani** [comment]
I guess we should go with view pager for now and we can shift to recycler views later

**Comment by Mridul97** [comment]
yes that will be good what all will be the powers of admin ?

**Comment by vaibhavchellani** [comment]
to view amount of people who have taken tokens , cancelling the queue , rest of the info is in admin panel screen, have a look

**Comment by Mridul97** [comment]
Where is the admin panel screen ? it is not there in the layouts above

**Comment by ghost** [comment]
the second layout is the admin panel screen I think

**Comment by Mridul97** [comment]
I think that is the user profile page there are two options also create queue and admin panel in that layout

**Comment by Rikki407** [comment]
@vaibhavchellani what is the logic behind showing the three different tokens?
Can't we just show the position in the queue that the user might be getting?

**Comment by vaibhavchellani** [comment]
so the second layout is the sidebar I forgot to mention , admin panel will include the option to edit what user entered while creating the queue , no of people currently in queue ,option of cancelling the queue

**Comment by vaibhavchellani** [comment]
I haven't created the admin panel screen yet

**Comment by vaibhavchellani** [comment]
one has to know which token is going on presently and also when his chance arrives he should be able to see if his token is next or next to next

**Comment by Rikki407** [comment]
but that can be known just by his position in the queue , whether he is standing in the 2nd position of the queue......you yourself said that the token will not be having any order so knowing which token number is going to precede him is not going to help him.....I think just displaying the position is more than sufficient

**Comment by Rikki407** [comment]
I mean thats what we do in real life.....we don't care who is standing before us....we just want to how many people are standing before us :)

**Comment by vaibhavchellani** [comment]
@IshaAgarwal98 @Mridul97 @Jasmeet1709 @Rikki407 @YashishDua 
this is the new design , we can change the layouts this is just rough (just a basic starting point )
![46c99d7e-f02e-401e-a41d-4223ea555f98](https://user-images.githubusercontent.com/19390504/31501431-e0a82d74-af87-11e7-9f0d-a8a2f9a25b6b.jpg)
credits : @IshaAgarwal98
